### PR TITLE
[incubator/haproxy-ingress] Add default-ssl-certificate flag configuration

### PR DESCRIPTION
**Pull request by Scukerman**
**What this PR does / why we need it**:
Adds a default-ssl-certificate flag configuration

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes none

**Special notes for your reviewer**:
none

### CODE REVIEW

**Comment by Scukerman** [comment]
checking CLA

**Review by xianlubird — COMMENTED** [review]
Pls fix it

File: incubator/haproxy-ingress/values.yaml [hunk 1]
```diff
@@ -31,6 +31,13 @@ controller:
     tag: "v0.5-beta.3"
     pullPolicy: IfNotPresent
 
+  defaultSslCertificate:
```
Comment (xianlubird) [inline incubator/haproxy-ingress/values.yaml:34]: This section should be added in READNE.md file .

**Review by xianlubird — COMMENTED** [review]
Pls fix it

File: incubator/haproxy-ingress/values.yaml (hunk 1 above)
Comment (Scukerman) [reply incubator/haproxy-ingress/values.yaml:34]: done. I also have squashed the commits.

**Pull request merged**